src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -362,7 +362,8 @@ protected boolean canBeInstantiated() {
 				.description("A location in a <a href='#world'>world</a>. Locations are world-specific and even store a <a href='#direction'>direction</a>, " +
 						"e.g. if you save a location and later teleport to it you will face the exact same direction you did when you saved the location.")
 				.usage("")
-				.examples("")
+				.examples("teleport player to location at 0, 69, 0",
+						  "teleport player to location at 0, 69, 0 in world 'world'")
```

Comment:
Changing an example
```suggestion
						  "set {home::%uuid of player%} to location of the player")
```

Reply:
You can always add more too!

Reply:
Oh my, now the pipeline starts to fail. If I cannot find a reason, I'll remove the example. 😇

Reply:
Nah don't worry, it's not due to you
Paper made some changes that break one of our tests, it'll be fixed soon

Reply:
I've found 12 errors, none of them seems to be related to this change:
* Getter in has been deprecated,
* Some EULAs issues,
* No key layers in MapLike...

Reply:
Yeah you can just ignore it